Dockerfile

```diff
@@ -2,37 +2,43 @@
 # Pull official python docker image
 FROM python:3.8.5-buster as base
 
-# The command below will run at build time and installs poetry 
-RUN pip install poetry
-
 # This is the active directory where commands will execute
-COPY . /src
 WORKDIR /src
-RUN poetry install
+COPY poetry.lock pyproject.toml /src/
+
+# Production Stage
+FROM base as production
+RUN pip install poetry \
+    && poetry config virtualenvs.create false
+RUN poetry install --no-root --no-dev
+COPY . /src/
+ENTRYPOINT gunicorn --bind 0.0.0.0:${PORT} wsgi:app
```

Comment:
It's worth noting this relies on the user setting the PORT environment variable, but that's not mentioned in your docs or set up by default anywhere. The nicest solutions would probably be either:
* set a default in your Dockerfile (`ENV PORT=8000`) which an explicit environment variable at runtime can override
* set a default value that's passed in through your docker-compose file